src/main/java/org/reactome/release/qa/check/MissingEditorialAttributeCheck.java

```diff
@@ -0,0 +1,99 @@
+package org.reactome.release.qa.check;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.schema.GKSchemaClass;
+import org.gk.schema.InvalidAttributeException;
+import org.reactome.release.qa.annotations.SliceQACheck;
+import org.reactome.release.qa.common.AbstractQACheck;
+import org.reactome.release.qa.common.QAReport;
+
+/**
+ * This QA check was requested by Lisa.
+ * Description:
+ * "Weekly QA and release QA: Report new events in
+ * slice (those where stableID released attribute
+ * is != True) and edited, reviewed, or authored
+ * attribute is null."
+ * @author sshorser
+ *
+ */
+@SliceQACheck
+public class MissingEditorialAttributeCheck extends AbstractQACheck {
+
+	private static final String YES = "YES";
+	private static final String NO = "NO";
+	private static Set<String> reportedEntities = new HashSet<>();
+
+	@Override
+	public QAReport executeQACheck() throws Exception
+	{
+		GKSchemaClass eventClass = (GKSchemaClass) this.dba.getSchema().getClassByName(ReactomeJavaConstants.Event);
+		QAReport report = new QAReport();
+		report.setColumnHeaders("DB_ID", "Name", "Edited?", "Authored?", "Reviewed?", "Created");
+		Collection<GKSchemaClass> classesToCheck = (Collection<GKSchemaClass>) eventClass.getSubClasses();
+		// make sure Event is in there with its subclasses.
+		classesToCheck = new ArrayList<>(classesToCheck);
+		classesToCheck.add(eventClass);
+		for (GKSchemaClass schemaClass : classesToCheck)
+		{
+			Collection<GKInstance> eventInstances = (Collection<GKInstance>) this.dba.fetchInstancesByClass(schemaClass);
+			for (GKInstance eventInstance : eventInstances)
+			{
+				String[] line = getReportLine(eventInstance);
+				if (line.length > 0)
+				{
+					report.addLine(line);
+				}
+			}
+		}
+		return report;
+	}
+
+	/**
+	 * Returns an array of Strings that will be added as a line to the report - IFF the eventInstance meets the criteria:
+	 * StableIdentifier is NOT released; Any of: authored, edited, reviewed are null.
+	 * @param eventInstance
+	 * @return
```

Comment:
Not strictly necessary since the method is private, but the param and return tags could take descriptions.